auth.js

```diff
@@ -13,28 +13,34 @@ module.exports.authenticate = async (req, res, next) => {
             req.user = { id, uuid };
             next();
         } catch (e) {
+            console.error(e);
             switch (e.message) {
             case 'User not found':
                 return res.status(404).json({
-                    error: true,
-                    data: 'User not found',
+                    data: null,
+                    errors: [{ title: 'User not found' }],
+                    meta: {},
```

Comment:
is `meta` used somewhere? @silverAndroid

Reply:
I added it to follow this: http://jsonapi.org/format/#document-structure

Reply:
but if we don't need this, I can remove it.
@BrandonDanis thoughts?

Reply:
I'd say to only add it when necessary

Reply:
I agree, maybe only include if necessary